src/v3/plugins/github/createGraph.test.js

```diff
@@ -2,11 +2,16 @@
 
 import {createGraph} from "./createGraph";
 import {GraphView} from "./graphView";
+import {RelationalView} from "./relationalView";
+import type {GithubResponseJSON} from "./graphql";
 import cloneDeep from "lodash.clonedeep";
 
 function exampleGraph() {
-  const data = cloneDeep(require("./demoData/example-github"));
-  return createGraph(data);
+  const data: GithubResponseJSON = cloneDeep(
```

Comment:
We could make this robust by creating a JavaScript module that exports

```js
// @flow

import cloneDeep from "lodash.clonedeep";

export function exampleGithub(): GithubResponseJSON {
  return cloneDeep(require("./example-github"));
}
```

so that clients don’t have to remember to (a) annotate the type and
(b) clone the raw value (which is stored in the `require` cache and
shared across all clients!).

You might also want to expose functions to construct the relational view
and/or graph from this example data, just for convenience.

Would you like to do that in a standalone commit prior to this one? (Or
after it, if you prefer.)

Reply:
I'd like to do it after this commit. Created an issue to track it: https://github.com/sourcecred/sourcecred/issues/430